SQL/New_patches/2020-02-10_NewModulePermissions.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO permissions (code,description,categoryID) VALUES
+('imaging_quality_control_view','Imaging Quality Control: View module',(SELECT ID FROM permissions_category WHERE Description='Permission'));
+INSERT INTO permissions (code,description,categoryID) VALUES
+('behavioural_quality_control_view','Behavioural Quality Control: View module',(SELECT ID FROM permissions_category WHERE Description='Permission'));
+INSERT INTO permissions (code,description,categoryID) VALUES
+('survey_accounts_view','Survey Accounts: View module',(SELECT ID FROM permissions_category WHERE Description='Permission'));
```

Comment:
This patch should also give the new permission to users who had the previous permissions that they replaced, so that access levels aren't changed.